src/locale/defaults.h

```diff
@@ -231,6 +249,9 @@
 #ifndef D_STR_AIRFLOW
 #define D_STR_AIRFLOW "Air Flow"
 #endif  // D_STR_AIRFLOW
+#ifndef D_STR_NATURALFLOW
+#define D_STR_NATURALFLOW "Natural Flow"
+#endif  // D_STR_NATURALFLOW
```

Comment:
[Q] What does this feature do? If we can, we want to reuse an existing term/word/name etc to reduce translation effort and to reduce the space required for the strings.

Reply:
I used Mitsubishis terminology from the remote and from the user’s manual. In this way it’s consistent for the next (Mitsubishi) user in line.
The functionality is that it just open the airflow from outside without heating or cooling.

Reply:
Would `kFreshStr` be an acceptable alternative? If not, leaving this as is fine. Same argument as Econo vs Ecocool, except "Natural Flow" is even larger/more space etc.

Reply:
OK, done for both Ecocool and Natural Flow